app/components/application-flow-step-link.js

```diff
@@ -20,22 +54,26 @@ export default class ApplicationFlowStepLinkComponent extends Component {
   }
 
   get isPreviousStep() {
+    // If subsidy is submitted, all the steps are previous
+    if (this.isSubmitted) return true;
```

Comment:
A previous step is a step already filled in or skipped. So a step that comes before the current active step. The meaning of that line is basically: The whole form isSubmitted so that means automatically that all steps are previous steps

Reply:
Will take care of the alert alignment :+1: